wwwapp/management/commands/populate_with_test_data.py

```diff
@@ -66,8 +66,8 @@ def fake_user(self) -> (User, UserProfile):
     """
     Creates and returns a fake random article with a random edit history
     """
-    def fake_article(self, users) -> Article:
-        article = Article(name=self.fake.uri_page(),
+    def fake_article(self, users, sequence) -> Article:
+        article = Article(name=self.fake.uri_page() + f'{sequence:04}',
```

Comment:
Are you sure the server is running a new-enough version of Python to support f-strings?